lib/encoder/include/MAE3.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "Encoder.h"
+#include "PwmIn.h"
+
+namespace Encoder {
+class MAE3 final : public Encoder {
```

Comment:
What's the MAE3 encoder used for? I didn't see it in the [actuator spreadsheet](https://docs.google.com/spreadsheets/d/1uotN7IJOHvdBjDnM055b1TbPaHrhUfwSUTXPCu_hMmY/edit#gid=0)

Reply:
its no longer used. we've replaced it with aeat6012, but its nice to have the driver